### ISSUE TITLE
New feature: Wish Estimator

### ISSUE THREAD

**Issue by alexpado** [MadeBaruna/paimon-moe#143]
Hi !

_First and foremost, as a disclaimer, I'm the developer of the **pretty well unknown** website [genshin-wish.akio.ovh](genshin-wish.akio.ovh)._

As your website has way more audience and a cleaner look, I'd like to work with you to integrate the tool I've coded into your website (and nuke what I've done, cause, yeah, the design is pretty lame).

The main highlight of the estimator, is that you can estimate how many pity you can get by filling multiple values, but don't give any probabilities.

You can even add daily commissions reward, packs and starglitter use (to buy more wishes). In the next update I was going to add Abyss & BP rewards and first pack purchase bonuses.

Do you think it would be possible to see such a tool integrated in your website ? You can also see the source code here: [alexpado/genshin-wish-calculator](https://github.com/alexpado/genshin-wish-calculator)

I'd like to start working on that, but there 2 major issues:

- I don't have any experience with Svelte, but I'd like to learn it. (And seems to have few similarities with Vue.js)
- I don't want to impose my vision, nor force your hand on that matter

This may be a very long issue, I'm sorry !

**Comment by MadeBaruna** [comment]
Yeah if you want you can make a PR! You might want to use this https://github.com/MadeBaruna/paimon-moe/blob/main/src/routes/calculator/_fates.svelte for references
